config/crds/apis.kcp.dev_apibindings.yaml

```diff
@@ -112,12 +112,17 @@ spec:
                             is bound to this API.
                           minLength: 1
                           type: string
+                        hash:
+                          description: hash is a hash of the APIResourceSchema's spec.
+                          minLength: 1
+                          type: string
```

Comment:
can you specify in the API go docs how hash and UID relate? What is the equality used by the controllers?

Reply:
Do you want to just remove hash for now?

Reply:
yes, would remove if it is not used

Reply:
done